cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -73,6 +121,12 @@ public static ClientAndServer server(int port) throws IOException {
         return new ClientAndServer(port);
     }
 
+    public static Header getAuthHeader() {
```

Comment:
If we are using this method in both the operator and the tests it might be safer to move it in with the other common CC stuff in the operator-common module?

Reply:
I was having  trouble reducing this since the CruiseControlApiImpl and  MockCruiseControl use different header objects, HTTPHeader and Header respectively. What I ended up doing was moving the core  logic to CruiseControlApiImpl and calling it from MockCruiseControl, casting HTTPHeader to Header where necessary. Does this look better?